src/App.js

```diff
@@ -6,13 +6,17 @@ import { Register, Login } from './components/auth';
 import Home from './components/home/home.jsx';
 import Navbar from './components/navbar/navbar';
 import accountSettings from './components/personalSettings/personalSettings.jsx';
+import formBuilder from './components/formBuilder/formBuilder';
+import forms from './components/forms/forms';
 
 function App() {
   return (
     <>
       <Navbar />
       <Switch>
         <Route exact path='/register' component={Register} />
+        <Route exact path='/forms' component={forms} />
+        <Route exact path='/form-builder' component={formBuilder} />
```

Comment:
you need to be consistent!
if all other routes, use PascalCase components (Register, Login, Home)
then these 2 needs to be PascalCase as well ->  Forms, FormBuilder